nntrainer/layers/pooling2d_layer.h

```diff
@@ -132,7 +107,9 @@ class Pooling2DLayer : public Layer {
    */
   void setBatch(InitLayerContext &context, unsigned int batch) override {
```

Comment:
@zhoonit with #1543, these interfaces can be cleaned up.

Reply:
Yeah, I'll stage this until the connection fixes are made, we might end up need this.